ci/tests/puppeteer/cas.js

```diff
@@ -288,12 +292,21 @@ exports.waitFor = async (url, successHandler, failureHandler) => {
         });
 }
 
+exports.getGradleCmd = async () => {
```

Comment:
You should most likely move this function inside the one that needs it. No need to export it publicly. I dont think anything else would ever want to use this, and the app itself at some point will run as an executable without Gradle altogether.